sda-commons-server-weld-example/build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
   api project(':sda-commons-server-dropwizard')
   api project(':sda-commons-server-weld')
 
-  testImplementation project(':sda-commons-server-weld-testing')
+  testImplementation project(':sda-commons-server-testing')
+  testImplementation 'org.jboss.weld:weld-junit5:2.0.2.Final'
```

Comment:
We've introduced "weld-junit5" to [server-weld-testing in version 6](https://github.com/SDA-SE/sda-dropwizard-commons/blob/6.0.0-rc.1/sda-commons-server-weld-testing/build.gradle). I'd to the same in this PR to prepare the change.

Our customer has specifically asked for maintaining a version of this library as convenience for the users of sda-dw-commons.

Reply:
We will add a managed version of weld-junit5 in the next major release of sda-dropwizard-commons.